frontend/views/utils/vSafeHtml.js

```diff
@@ -29,8 +30,12 @@ export const defaultConfig = {
 
 const transform = (el, binding) => {
   if (binding.oldValue !== binding.value) {
-    const config = { ...defaultConfig, ...(binding.arg || {}) }
-
+    let config = defaultConfig
+    if (binding.arg === 'a') {
+      config = cloneDeep(config)
+      config.ALLOWED_ATTR.splice(config.ALLOWED_ATTR.length, 0, 'href', 'target')
```

Comment:
This `splice` call doesn't seem to remove any element, only to add `href` and `target`. So did you mean the following?
```suggestion
      config.ALLOWED_ATTR.push('href', 'target')
```

Reply:
Lol yes, for some reason I thought `.push` only accepted one argument, learn something new every day. Thanks! :)